scripts/db_connections.py

```
@@ -0,0 +1,65 @@
+
+
+def load_config(name: str) -> dict:
+    """Load a configuration file by name (without extension)."""
+    path = CONFIG_DIR / f"{name}.json"
+    if not path.exists():
+        raise FileNotFoundError(f"Config {name} not found")
+    with path.open() as f:
+        return json.load(f)
+
+
+Drop the manual `_conn` state and `close()` in favor of sqlite3’s built-in context manager. This halves the code and removes error-prone state. For example:
```

Ruff (0.11.9)

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Got unexpected token `

---

SyntaxError: Got unexpected token `

---

SyntaxError: Got unexpected token `

---

SyntaxError: Got unexpected token `

---

SyntaxError: Expected a statement

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Got unexpected token ’

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an identifier, but found a keyword 'in' that cannot be used here

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

Line too long (156 > 122)

(E501)